### ISSUE TITLE
The mesh position should be set *before* calling addMesh, not inside it

### ISSUE THREAD

**Issue by DavidBruant** [anthill/city-core#87]
The mesh should know its own position. Move this to client.js
https://github.com/DavidBruant/bordeaux3d-core/commit/fe2d7a9923d33ae40492e140d75deaa3901f9fb4#diff-44b390af6d21196c55fb50f726184ffcR100

**Comment by iOiurson** [comment]
I agree, but in the actual state of what the AddMesh button does (positioning the cube in front of the camera), this would need the camera to be exported by index.js. I'm not sure this is a good thing.
Also, in the future, we need to think about how the user will be able to position its own meshes.

**Comment by DavidBruant** [comment]
> this would need the camera to be exported by index.js

[It is ;-)](https://github.com/anthill/city-core/blob/42347d0587f3fe53666695b62b4324741fa494e9/front/src/index.js#L130)
And it's been exported for other reasons https://github.com/anthill/city-core/blob/42347d0587f3fe53666695b62b4324741fa494e9/core-client/client.js#L35-L51

**Comment by DavidBruant** [comment]
> we need to think about how the user will be able to position its own meshes.

True. For now, we can expect the position is encoded in the 3ds.

**Comment by DavidBruant** [comment]
Sorry, I meant the mesh, not the 3ds.

**Comment by iOiurson** [comment]
Okay, i wasn't working with the latest code you wrote, so i didn't have the camera exported. Review and merge #88 if you're ok with it, and i'll pull latest version of core-separation to fix this.

**Comment by DavidBruant** [comment]
It's a small change so push it as part of #88 (but in its own commit which references this issue).
I'm working on something else today, so I'll merge this week-end or on Monday.

As soon as #88 is merged, we should move the master branch to the top of core-separation, because the biggest part of the work is done and the API has proven to work with `core-client`.
